### DISCUSSION TITLE
How can I configure the Server to use a password ?

### DISCUSSION THREAD

**Discussion by MeshuggahMischell** [gioddiggi/sons-of-the-docker#2] [Q&A] [answered]
Hello, how can I access the Server to change the configuration to use a password to access the Server ?
And how should I restart the Server then ?

**Comment by gioddiggi** [comment, marked as answer]
Hi @MeshuggahMischell, sorry for the delay of the answer.

You can just fill the field `Password` inside your server configuration file (`dedicatedserver.cfg`) located at `/root/.wine/drive_c/users/root/AppData/LocalLow/Endnight/SonsOfTheForestDS` inside the container (or you can build container image again once you have edited `cfg/dedicatedserver.cfg`).
If you want to restart the server, you might consider to restart the container. 

I planned to externally map two volumes, one for saves, one for configurations, I will prob implement it on the next release.



EDIT: Starting from version `v0.8.0` you can now add `-v <CONFIG_FOLDER_PATH>:/srv/sons-of-the-docker/config` to your `docker run` command, so you can map your config file on the host.

**Comment by gioddiggi** [comment]
Closing.
